stl/inc/bitset

```diff
@@ -400,7 +401,18 @@ public:
     }
 
     _NODISCARD bool all() const noexcept {
-        return count() == size();
+        if _CONSTEXPR_IF (_Bits == 0) {
+            return true;
+        }
+        for (size_t _Wpos = 0; _Wpos < _Words; ++_Wpos) {
+            if (_Array[_Wpos] != ~static_cast<_Ty>(0)) {
+                return false;
+            }
+        }
+        if _CONSTEXPR_IF ((_Bits % _Bitsperword) == 0) {
+            return _Array[_Words] == ~static_cast<_Ty>(0);
+        }
+        return _Array[_Words] == (static_cast<_Ty>(1) << (_Bits % _Bitsperword)) - 1;

```

Comment:
After `if (condition) { stuff; return stuff; }` we often prefer to avoid `else` unnecessarily indenting the control flow. Here, as this can expand to `if constexpr`, I think we should have an `else` - that way, there's a (minor, theoretical) throughput improvement, and we usually use `else` with `if constexpr`. It would also slightly increase the symmetry of the return statements; I wouldn't object to an `else` even with a plain `if` here.

Reply:
Ruled out with having logical or instead.